mathesar/api/ui/permissions/users.py

```diff
@@ -11,7 +11,7 @@ class UserAccessPolicy(AccessPolicy):
         },
         # Only superusers can create users
         {
-            'action': ['create'],
+            'action': ['create', 'password_reset'],
```

Comment:
Users should be able to reset their own passwords, it shouldn't just be superusers.

Reply:
Okay yes, you're correct. We do need a reset password flow for users later (using email to verify), but that's not in scope right now.

Reply:
Created #2052 for the password change flow